### DISCUSSION TITLE
Problems with national meta.yml file

### DISCUSSION THREAD

**Discussion by SDGArmenia** [open-sdg/open-sdg#1949] [Q&A]
Hi Everyone,
 
Can anybody help me find the problem?

When we try to divide the text by paragraph, the system prints the translation key meta.calculations-15-3-1-a , otherwise prints all together, .

For example in meta.yml file for this indicator the text must be divided by paragraph,

definitions-15-3-1-a: "Soil erased by wind - these soils decompose under the influence of wind (wind erosion-deflation).
Soil erased by water - these soils degrade under the influence of water (water erosion).
Salinized -these soils contain easily soluble amounts of toxic salts  in their profile for agricultural crops (salt marshes, salinized soils, alkalis).
Secondary salinized - where the process of accumulation of harmful salts for agricultural crops is conditioned by wrong human economic activity (irrigation equipment, increase of groundwater level as a result of violation of regimes, irrigation with mineralized water, etc.)
Alkalinized - these soils contain a large amount of exchangeable sodium (sometimes magnesium) when absorbed, in the alluvial horizon.
Unsettled soil - damaged by entrails extraction (quarries, open pit mines, etc.), as it were, soils that are covered with industrial wastes, mining chemicals, etc. These are decommissioned lands that have not been reclaimed.
Hyper - moist soils are completely saturated with water.
Rocky - These are the soils that contain stones of different sizes in their profile.
Contaminated with waste - These are the soils (areas) that are polluted with various pollutants (industrial, construction, domestic, etc.)
Swamped - these are the soils that have been formed under the influence of special moist (swamp) vegetation in conditions of excess surface or groundwater moisture. 
Deserted - these are the soils where the features (indicators) typical of desert landscapes appear. Desertification is the degradation of lands (territories)."

link to data repository  -   https://github.com/armstat/sdg-data-armenia
link to the file with the issue  -    https://github.com/armstat/sdg-data-armenia/blob/develop/translations/en/meta.yml
link to the site   -  https://sdg.armstat.am/15-3-1-a/                       - here prints all paragraphs together

Тhank you in advance!

**Comment by brockfanning** [comment]
@SDGArmenia If possible could you post some links? In particular it would be good to have:
- link to data repository
- link to the file with the issue
- link to the site

**Comment by brockfanning** [comment]
@SDGArmenia Thank you for adding the links! Can you try adding a blank line where you would like a new paragraph? For example, instead of this:

```
Soil erased by wind - these soils decompose under the influence of wind (wind erosion-deflation).
Soil erased by water - these soils degrade under the influence of water (water erosion).
```

...try adding a blank line like this:

```
Soil erased by wind - these soils decompose under the influence of wind (wind erosion-deflation).

Soil erased by water - these soils degrade under the influence of water (water erosion).
```

The "markdown" formatter should hopefully interpret that correctly and treat them as two separate paragraphs.

**Reply by brockfanning** [reply]
@SDGArmenia Alternatively, if you would like them to be in a bulleted list, you could do this:

```
* Soil erased by wind - these soils decompose under the influence of wind (wind erosion-deflation).
* Soil erased by water - these soils degrade under the influence of water (water erosion).
```

Markdown will render that as a bulleted list, like so:

* Soil erased by wind - these soils decompose under the influence of wind (wind erosion-deflation).
* Soil erased by water - these soils degrade under the influence of water (water erosion).

Markdown has a few little tricks like this. For more information on Markdown see here: https://www.markdownguide.org/cheat-sheet/
